### PR TITLE
fix: Fix aria-label for BE header search

### PR DESCRIPTION
**Pull request by tjuanitas**
Matches the search inputs `aria-label` and `placeholder` messages

### CODE REVIEW

File: src/elements/common/header/Header.js [hunk 1]
```diff
@@ -33,10 +34,11 @@ const Header = ({ isHeaderLogoVisible = true, view, isSmall, searchQuery, onSear
             {isHeaderLogoVisible && <Logo isSmall={isSmall} url={logoUrl} />}
             <div className="be-search">
                 <input
-                    aria-label="search"
+                    aria-label={formatMessage(messages.searchPlaceholder)}
+                    data-testid="be-search-input"
                     disabled={!isFolder && !isSearch}
                     onChange={search}
-                    placeholder={intl.formatMessage(messages.searchPlaceholder)}
+                    placeholder={formatMessage(messages.searchPlaceholder)}
```
Comment (QingyuChai) [inline src/elements/common/header/Header.js:41]: Since it's being used in two places, I'd prefer we use a variable, but this is fine as well

File: src/elements/common/header/__tests__/Header.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    test('renders Logo component when isHeaderLogoVisible is passed', () => {
+        const wrapper = getWrapper({ isHeaderLogoVisible: true });
+        expect(wrapper.find('Logo').exists()).toBe(true);
+    });
+
+    test('renders matching values for aria-label and placeholder attributes', () => {
+        const wrapper = getWrapper();
+        const inputProps = wrapper.find('[data-testid="be-search-input"]').props();
+        expect(inputProps['aria-label']).toBe(inputProps.placeholder);
```
Comment (QingyuChai) [inline src/elements/common/header/__tests__/Header.test.js:20]: minor: you're using using `inputProps['aria-label']`, but `inputProps.placeholder`. They're both fine, but I think we should be consistent

Comment (tjuanitas) [reply src/elements/common/header/__tests__/Header.test.js:20]: yeah this was an eslint thing. it prefers dot notation but it can't be used for the aria-label

Comment (QingyuChai) [reply src/elements/common/header/__tests__/Header.test.js:20]: oh that's interesting, thanks for the info

File: src/elements/common/header/__tests__/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
```
Comment (JChan106) [inline src/elements/common/header/__tests__/Header.test.js:2]: Think we're trying to switch away from enzyme, can we use testing-library instead

File: src/elements/common/header/Header.js [hunk 4]
```diff
@@ -45,4 +49,5 @@ const Header = ({ isHeaderLogoVisible = true, view, isSmall, searchQuery, onSear
     );
 };
 
+export { Header as HeaderBase };
```
[the remark this answers is not in the document]
Comment (tjuanitas) [reply src/elements/common/header/Header.js:52]: this is so the base component can be tested without the intl injection stuff

File: src/elements/common/header/__tests__/Header.test.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        formatMessage: jest.fn().mockImplementation(message => message.defaultMessage),
+    };
+
+    const getWrapper = props => shallow(<Header intl={intl} {...props} />);
+
+    test('renders Logo component when isHeaderLogoVisible is passed', () => {
+        const wrapper = getWrapper({ isHeaderLogoVisible: true });
+        expect(wrapper.find('Logo').exists()).toBe(true);
+    });
+
```
Comment (karelee7) [inline src/elements/common/header/__tests__/Header.test.js:16]: Should we also add a test for the negative case above? (i.e. doesn't render logo when `isHeaderLogoVisible` is not passed or falsey)

**Review by karelee7 — APPROVED** [review]
Looks good to me!

**Review by QingyuChai — APPROVED** [review]
